src/main/webapp/pages/admin/index.html

```diff
@@ -1,10 +1,28 @@
 <!DOCTYPE html>
-<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<html lang="en"
+      xmlns:th="http://www.thymeleaf.org"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout">
```

Comment:
Don't need \`xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"\` it is used for doing complicated templating on the main site that I really wish was not there.